flixel/math/FlxPoint.hx

```diff
@@ -419,7 +419,7 @@ class FlxPoint implements IFlxPooled
 	 * Function to get a `FlxVector` from this `FlxPoint`
 	 * @since 4.3.0
 	 */
-	@:deprecated
+	@:deprecated("The `toVector` method is deprecated, use `cast(myPoint, FlxVector)` or `FlxVector.get` instead")
```

Comment:
I'd actually consider a safe cast bad practice here, that has unnecessary runtime overhead. There's no explicit casts needed to trigger the _implicit_ casts of abstracts.

Reply:
scratch that last comment, I don't like changing the constructor of FlxVector

Reply:
Can't it just say "use implicit casts"?

It's not clear to me how changing the constructor would be helpful? Typically you'd just use assignments or type checks to trigger implicit casts.

```haxe
var vector:FlxVector = someFlxPoint;
vector.someVectorMethod();

(someFlxPoint : FlxVector).someVectorMethod();
```

Reply:
```The `toVector` method is deprecated, FlxPoints can be casted to FlxVectors implicitly```
???

Reply:
Sure. :)